### DISCUSSION TITLE
Issue with imports

### DISCUSSION THREAD

**Discussion by goofylawz** [Cyfrin/foundry-full-course-cu#3078] [Q&A] [answered]
Good morning frens, 

Another project, another issue with imports. Even though I was able to fix all my previous issues, every once in a while when I start a new project, I stumble upon a problem with imports.

I have the following code in my file:
```

pragma solidity ^0.8.18;

import {ERC20Burnable, ERC20} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";

contract DecentralizedStableCoin is ERC20Burnable, Ownable {
    error DecentralizedStableCoin__MustBeMoreThanZero();
    error DecentralizedStableCoin__BurnAmountExceedsBalance();
    error DecentralizedStableCoin__NotZeroAddress();

    constructor(
        address initialOwner
    ) ERC20("DecentralizedStableCoin", "DSC") Ownable(initialOwner) {}

    function burn(uint256 _amount) public override onlyOwner {
        uint256 balance = balanceOf(msg.sender);
        if (_amount <= 0) {
            revert DecentralizedStableCoin__MustBeMoreThanZero();
        }
        if (balance < _amount) {
            revert DecentralizedStableCoin__BurnAmountExceedsBalance();
        }
        super.burn(_amount);
    }

    function mint(
        address _to,
        uint256 _amount
    ) external onlyOwner returns (bool) {
        if (_to == address(0)) {
            revert DecentralizedStableCoin__NotZeroAddress();
        }
        if (_amount <= 0) {
            revert DecentralizedStableCoin__MustBeMoreThanZero();
        }
        _mint(_to, _amount);
        return true;
    }
}
```

The two imports are highlighted in red, and when I try cmd+click, I cannot open the contracts, even though OpenZeppelin contracts were successfully installed. I'm able to find the files manually. I also tried to locate the OpenZeppelin-contracts folder in the terminal, got some advice through AI, and everything seems fine. The project was compiled successfully, but those imports are still red. 

![image](https://github.com/user-attachments/assets/18a3e59c-f869-49fb-be82-db3a2643d915)
![image](https://github.com/user-attachments/assets/d15933d8-de19-4673-a7c1-a9a6f4bc440f)

I'm pretty sure this problem is not a blocker for my course, but i'm interested to know what could be the source of this issue so I can fix it.

Thanks in advance for any suggestions/tips/recommendations.

**Comment by EngrPips** [comment, marked as answer]
Hello @goofylawz, Please close your `vscode` and reopen it. Let's see if the imports still have the red underline.

**Reply by goofylawz** [reply]
Ser, wtf is this *** :DDD It worked! Thank you frend!

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.
